client/src/document/index.scss

```
@@ -106,7 +106,7 @@
   }
 
   img {
-    background: #fff;
+    background: transparent;
```

Let's use `--background-primary` instead to be safe:

```suggestion
    background: var(--background-primary);
```